dts/bindings/led/dac-leds.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2018, Linaro Limited
+# Copyright (c) 2025, Prevas A/S
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Group of DAC-controlled LEDs.
+
+  Each LED is defined in a child node of the dac-leds node.
+
+  Here is an example which defines an LED in the node /leds:
+
+  / {
+    leds {
+      compatible = "dac-leds";
+      led_0 {
+        dac = <&dac1>;
+        channel = <0>;
+        resolution = <12>;
+      };
+    };
+  };
```

Comment:
I would use dts indentation, 8 spaces, not a blocker but I think its neater. Two spaces before `  / {` then 8 spaces

Reply:
It is more or less copy paste from `gpio-leds.yaml` so I did not really consider the indentation. `leds` actually needs to be indented that should be fixed.

Is it allowed to use tabs in comments (I think copy pasting example into a device tree source file looks as if indentation is ok, but it isn't since it is 8 spaces instead of tabs)?

Reply:
Its not indented to be copied directly to the devicetree, just to look like a devicetree snippet :)

Reply:
I revisited `gpio-leds.yaml` and it actually uses tabs. My editor converted them to spaces. I have changed them back to tabs.

Reply:
Never mind. `check_compliance.py` wont allow tabs after space. I changed indent to 8 spaces.